app/assets/v2/js/pages/new_bounty.js

```diff
@@ -271,6 +273,21 @@ const tokenAuthAlert = (isTokenAuthed, tokenName) => {
   }
 };
 
+
+const updateViewForToken = (token_name) => {
+
+  if (token_name == 'ETC') {
+    $('#featuredBounty').prop('checked', false);
```

Comment:
is not the opposite?  true for ETC

Reply:
@octavioamu ah so that's taken care [here](https://github.com/gitcoinco/web/pull/5600/files#diff-bda00bc127f28e4f8ec2648a72152af7R33)

But pointing that out -> it makes sense to not even keep it there !  
removing it from there